### ISSUE TITLE
"Select Rows (with limit)" for large field ( text/ blob) cause sqlectron to freeze

### ISSUE THREAD

**Issue by terrywh** [sqlectron/sqlectron-gui#285]
"Select Rows (with limit)" query for large field ( text/ blob) cause sqlectron to freeze.

maybe add some feature like in heidisql (http://heidisql.com) to  "LEFT(`field`)" ?

**Comment by eMerzh** [comment]
yes.... i think it's somewhat related to #129 but in another fashion ...
not really sure how we can auto fix this ...

**Comment by terrywh** [comment]
query for the table schema, and do the ``LEFT(`field`)`` on field with length greater than 1024 ?

**Comment by maxcnunes** [comment]
@terrywh If you include a limit of 100 for that table, does it still freezing? If it still freezing than is not directly related to #129. But could be related to #174, although in this case is text/blob instead of JSON. The `LEFT` (for mssql, other clients use different keyword) command could definitely help. This looks a good and easy solution. But I would prefer to see this in a different menu item instead of including it in the "Select rows (with limit)". Could be "Select rows (with limit and fixed column size)". So would allow the users choose what works better for them.

Right now I'm focusing in make the app more stable fixing the more critical bugs. Although this can cause the app to freeze. It still can be avoided by adding the `LEFT` keyword manually. So it is something I don't intend to implement myself so soon. If you have interest in sending a PR for that. Please do, I can help with any development question.

**Comment by terrywh** [comment]
limit of 100 rows still freezing but does so for shorter time period.( I have those blob/binary fields contain image file range from 300KB ~ 2Mb)
